src/main/java/com/sap/oss/phosphor/fosstars/model/value/VersionRange.java

```diff
@@ -1,34 +1,32 @@
 package com.sap.oss.phosphor.fosstars.model.value;
 
 import com.fasterxml.jackson.annotation.JsonGetter;
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Objects;
 import java.util.Optional;
 
 /**
- * The class holds a start and end version.
+ * The class holds a version range.
  */
-@JsonInclude(JsonInclude.Include.NON_NULL)
 public class VersionRange {
 
   /**
-   * First vulnerable version (including).
+   * A start version.
    */
   private final String versionStart;
 
   /**
-   * Last vulnerable version (including).
+   * An end version.
    */
   private final String versionEnd;
 
   /**
-   * Create a version range with start and end version.
+   * Create a version range with start and end versions.
    * Both versions are including.
    * Start and end version may be null.
    *
-   * @param versionStart start version (including)
```

Comment:
As I understand. We do not have any way of deciding if the end or start versions is including or excluding. Atleast we have not implemented anything to address this. Do we not mention that as part of doc?

Reply:
> We do not have any way of deciding if the end or start versions is including or excluding.

No, the class doesn't control it. Even if we mention "including" or "excluding" in the docs, the caller is free to follow it or not.

> Atleast we have not implemented anything to address this

No, we haven't. To be honest, I am not sure that we need it.